config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph_hci.yaml

```diff
@@ -5,16 +5,14 @@ metadata:
   namespace: openstack
 spec:
   env:
-  - name: ANSIBLE_CALLBACKS_ENABLED
-    value: profile_tasks
-  - name: ANSIBLE_FORCE_COLOR
-    value: "True"
+    - name: ANSIBLE_CALLBACKS_ENABLED
+      value: profile_tasks
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
   networkAttachments:
   - ctlplane
   nodeTemplate:
     ansible:
-      ansiblePort: 22
-      ansibleUser: cloud-admin
       ansibleVars:
         ctlplane_dns_nameservers:
```

Comment:
These all are not required as this sample is using IPAM.

ctlplane_dns_nameservers
ctlplane_mtu
ctlplane_subnet_cidr
dns_search_domains

Reply:
Very true. I'll remove all of these now as well.